code/modules/projectiles/guns/projectile/automatic/atreides.dm

```diff
@@ -21,8 +21,8 @@
 	recoil_buildup = 1.2
 	one_hand_penalty = 5 //smg level
 	gun_tags = list(GUN_SILENCABLE, GUN_GILDABLE)
-	wield_delay = 0.5 SECOND
-	wield_delay_factor = 0.1 // 10 vig for instant
+	wield_delay = 0 SECOND
+	wield_delay_factor = 0
```

Comment:
Inherited from code/modules/projectiles/guns/projectile/automatic.dm, can be removed

Reply:
Same goes for all the rest too

Reply:
Just removed it. Tell me if i missed anything